.github/workflows/CI.yml

```diff
@@ -10,11 +10,19 @@ jobs:
       matrix:
         os: [macOS-latest]
       fail-fast: true
+    env:
+      FC: gfortran
+      GCC_V: 12
 
     steps:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - name: Install Dependencies MacOS
+      run: |
+        brew install gcc@${GCC_V} gcc@9
```

Comment:
@everythingfunctional what's the reason for `gcc@9` here?

Reply:
The fpm releases are still built with gcc@9, so you need to have that version of the runtime library installed for fpm to work (I think).